src/utils/jwk.ts

```diff
@@ -37,3 +37,26 @@ export const JWK = z.object({
   /** JWK "x5u" (X.509 URL) Parameter. */
   x5u: z.string().optional(),
 });
+
+/**
+ * Ensure key values are encoded using base64url and not just base64, as defined in https://datatracker.ietf.org/doc/html/rfc7517
+ *
+ * @see https://datatracker.ietf.org/doc/html/rfc7517
+ *
+ * @param key The key to fix
+ * @returns THe same input key with fixed values
+ */
+export function fixBase64EncodingOnKey(key: JWK): JWK {
+  const { x, y, e, n, ...pk } = key;
+  const removePadding = (encoded: string) =>
+    // eslint-disable-next-line no-div-regex
+    encoded.replace(/=/g, "").replace(/\+/g, "-").replace(/\//g, "_");
```

Comment:
You can use `removePadding` from `io-react-native-jwt`

Reply:
You can use `io-react-native-jwt` [0.6.4](https://github.com/pagopa/io-react-native-jwt/releases/tag/v0.6.4) which now exposes removePadding and addPadding

Reply:
done in https://github.com/pagopa/io-react-native-wallet/pull/31/commits/f1f2dc2325dd5784529d52bd5ddc0422892f5758